scripts/route_check.py

```diff
@@ -356,6 +356,12 @@ def filter_out_local_interfaces(keys):
     tbl = swsscommon.Table(db, 'ROUTE_TABLE')
 
     for k in keys:
+        if tbl.get(k)[0] == False:
+            # ISU2023101259067, if kernel route prefix is xxxx::yy/128, fpmsyncd will write ROUTE_TABLE:xxxx::yy without /128 to apll db
```

Comment:
What is the ISUXXX, please remove it from comment. Also, can you add a unit-test?